s12/flags/permissions/permissions.proto

```diff
@@ -14,4 +14,10 @@ extend google.protobuf.MethodOptions {
   //       option (google.api.http).get = "/users/v1/profiles:GetUser"
   //     }
   repeated string required_flags = 20100;
+  // A list of required scopes for the given method.
+  // For example:
+  //     rpc Getuser(Request) returns (Response) {
+  //       option (s12.flags.permissions.required_scopes) = "admin"
+  //     }
+  repeated string required_scopes = 20101;
```

Comment:
I have reached out to @bong-sc about dropping the `repeated` attribute from here. The reason being, we do not want to accidentally allow a user scope for an admin endpoint.

Reply:
Checked with @sc-peter offline, it's better to use `optional string` considering it's either an admin or a user endpoint.